notebooks/tutorials/utilities.ipynb

```diff
@@ -157,137 +157,76 @@
    "id": "76ee7e5e",
    "metadata": {},
    "source": [
-    "# Visualizations"
+    "# Coordinates"
```

Comment:
There is a separate tutorial file for coordinates already. Is there a reason for not putting this there?

Reply:
Because I didn't realize the coordinates tutorial existed! I added it to the Tutorials html page